app/agent/collector.scala

```diff
@@ -51,11 +51,14 @@ class CollectorAgent[T<:IndexedItem](val collectorSet: CollectorSet[T], sourceSt
         val marker = Markers.appendEntries((l.toMarkerMap ++ this.toMarkerMap ++ Map("duration" -> timeSpent)).asJava)
         previous.label match {
           case bad if bad.isError =>
-            log.error(s"Crawl of ${product.name} from $origin failed (${timeSpent}ms): NO data available as this has not been crawled successfuly since Prism started", error)(marker)
+            log.error(s"Crawl of ${product.name} from $origin failed (${timeSpent}ms): NO data available as this has not been crawled successfully since Prism started", error)(marker)
           case stale if stale.bestBefore.isStale =>
             log.error(s"Crawl of ${product.name} from $origin failed (${timeSpent}ms): leaving previously crawled STALE data (${stale.bestBefore.age.getStandardSeconds} seconds old)", error)(marker)
           case notYetStale if !notYetStale.bestBefore.isStale =>
             log.warn(s"Crawl of ${product.name} from $origin failed (${timeSpent}ms): leaving previously crawled data (${notYetStale.bestBefore.age.getStandardSeconds} seconds old)", error)(marker)
+
+          // this shouldn't happen, but we need to be exhaustive
+          case _ => log.warn(s"Crawl of ${product.name} from $origin failed (${timeSpent}ms)", error)(marker)
```

Comment:
This resolves this compile error:

```console
[info] compiling 53 Scala sources and 1 Java source to /Users/akash_askoolum/code/prism/target/scala-2.13/classes ...
[error] /Users/akash_askoolum/code/prism/app/agent/collector.scala:52:18: match may not be exhaustive.
[error]         previous.label match {
[error]                  ^
[error] one error found
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 6 s, completed 3 Oct 2022, 15:33:29
```

Not sure _why_ this has started to be reported though!